contracts/capacitors/HashChainCapacitor.sol

```diff
@@ -5,49 +5,172 @@ import "./BaseCapacitor.sol";
 
 /**
  * @title HashChainCapacitor
- * @notice This is an experimental contract and have known bugs
  * @dev A contract that implements ICapacitor and stores packed messages in a hash chain.
- * The hash chain is made of packets, each packet contains a maximum of 10 messages.
+ * The hash chain is made of packets, each packet contains a capped number of messages.
  * Each new message added to the chain is hashed with the previous root to create a new root.
  * When a packet is full, a new packet is created and the root of the last packet is sealed.
  */
 contract HashChainCapacitor is BaseCapacitor {
-    uint256 private _chainLength;
-    uint256 private constant _MAX_LEN = 10;
+    uint64 private constant _MAX_LEN = 10;
```

Comment:
Wheres the change for updating _MAX_LEN? there was also the comment informing we need to create packets in case length is being reduced.

Reply:
max length change was done by @tHeMaskedMan981  in #251 
https://github.com/SocketDotTech/socket-DL/blob/gasestimate/contracts/capacitors/HashChainCapacitor.sol#L14

Reply:
And the fix for reducing length change is in latest PRs with review fixes here:
https://github.com/SocketDotTech/socket-DL/pull/259/commits/68ea056e4638a5b1ac9efddda208a4de6e2c1e2f